tutorials/3d/csg_tools.rst

```diff
@@ -126,21 +127,21 @@ Create a ``CSGBox`` and adjust its dimension for the bed. Create another ``CSGBo
 
 .. image:: img/csg_bed_mat.png
 
-We will create another ``CSGCombiner`` and name it ``pillow`` as the child of  ``bed``. The scene tree should look like this:
+We will create another ``CSGCombiner``\ , named ``pillow``\ , as the child of  ``bed``. The scene tree should look like this:
```

Comment:
Is the `"\ "` (escaped space) required (here and below on line 157)?

Reply:
Nope, I fixed it.

Reply:
> Is the `"\ "` (escaped space) required (here and below on line 157)?

> Nope, I fixed it.

@akien-mga : So, I suppose only letters adjacent to the closing double-backtick (LISP :grinning:) / grave accent (linguistics) throw parsing errors?